modelator/src/artifact/tla_file_suite.rs

```diff
@@ -1,4 +1,5 @@
 use std::collections::BTreeSet;
+use std::iter::once;
```

Comment:
did you use it in the file?

Reply:
Nice catch, I was playing with it but ended up not using it.